ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -609,6 +618,11 @@ ORK_CLASS_AVAILABLE
  */
 @property (readonly, getter=isVertical) BOOL vertical;
 
+/**
```

Comment:
remove the setter here to remain consistent with the previous two property declarations?

Reply:
I added the getter with this format here to match the precedent of -isVertical and and -isSecureTextEntry in ORKAnswerFormat.

Reply:
Here are the two different signatures that is part of this PR -

1. @property (assign, getter=shouldHideSelectedValueLabel) BOOL hideSelectedValue;
2. @property (assign, getter=shouldHideSelectedValueLabel,setter=setHideSelectedValue:) BOOL hideSelectedValue;

Can we just stick with the signature in (1) ?